flow2ml/Data_Augumentation.py

```diff
@@ -146,4 +146,36 @@ def applyCrop(self,classPath):
           else:
             raise Exception("Cropping needs random parameter or list of coordinates.")
         except Exception as e:
-          print(f"Crop operation failed due to {e}")
\ No newline at end of file
+          print(f"Crop operation failed due to {e}")
+  
+  def applyScale(self,classPath):
```

Comment:
I think you should add “random” scenario also in scaling operation

Reply:
How exactly do I include that? Scale all images by some random ratio?